recipes/llvmlite/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "llvmlite" %}
+{% set version = "0.20.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "b2f174848df16bb9195a07fec102110a06d018da736bd9b3570a54d44c797c29" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - enum34
```

Comment:
`enum34` should have a selector of e.g. `py < 34`, or just `py2k`, to [match the `setup.py`](https://github.com/numba/llvmlite/blob/9088d9fc4ac18119e07c802def86d7fd73c3678d/setup.py#L161-L163) and because it's not packaged on higher versions of python. :)